.mocharc.js

```diff
@@ -0,0 +1,13 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const {mergeMochaConfigs} = require('./packages/build');
+const defaultConfig = require('./packages/build/config/.mocharc.json');
+
+module.exports = mergeMochaConfigs(
+  defaultConfig,
+  require('./packages/cli/.mocharc.js'),
+  // TODO: add Mocha configs for other packages
+);
```

Comment:
I am not sure about this design - at one hand it feels a bit like overkill for our current needs, on the other hand I dislike the version proposed in https://github.com/strongloop/loopback-next/pull/5011 where we were using the CLI mocha options directly. Maybe instead of a generic merge algorithm it would be enough to merge `timeout` and `require` entries only?

Also in #5011, `process.env.MOCHA_JOBS` was handled by CLI's mocha file - I don't think that's correct. I would expect that flag to be handled either at monorepo level or inside `packages/build/config/.mocharc.json`. The latter would be a bit more tricky because we don't want to enable `parallel` mode for everybody. Anyhow, we can discuss that elsewhere, it's out of scope of this PR.